<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSuccess</class>
 <widget class="QDialog" name="DialogSuccess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogSuccess</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Коэффициент успешности</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QDateTimeEdit" name="fromDateTimeEdit"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="toDateTimeEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="enterButton">
       <property name="text">
        <string>ВВОД</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>ЗАКРЫТЬ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSuccess</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enterButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSuccess</receiver>
   <slot>EnterClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>EnterClicked()</slot>
 </slots>
</ui>
